site/app/controllers/api/TokenController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+
+namespace app\controllers\api;
+
+use Symfony\Component\Routing\Annotation\Route;
+
+
+class TokenController extends AbstractApiController {
+
+    /**
+     *
+     * @Route("/token", methods={"GET"})
+     */
+    public function getToken() {
+        $this->core->getOutput()->renderJsonSuccess("test");
```

Comment:
I think is just setting up the basic router for the api, but how would you write a test?